### DISCUSSION TITLE
[Question][Hw6] number of variables

### DISCUSSION THREAD

**Discussion by sms0509** [prosyslab-classroom/cs424-program-reasoning#304] [Q&A]
In HW6, can I assume that there are exactly two variables for each testcase?

**Comment by sms0509** [comment]
and have names 'x' and 'y'?

**Comment by Hhro** [comment]
No, you can’t.
There are no constraint for the number and name of the variables.

**Comment by sms0509** [comment]
Thank you for the answer!
